addons/portal/models/mail_message.py

```diff
@@ -16,8 +16,10 @@ def portal_message_format(self):
 
     def _portal_message_format(self, fields_list):
         vals_list = self._message_format(fields_list)
+        note_id = self.env['ir.model.data'].xmlid_to_res_id('mail.mt_note')
         IrAttachmentSudo = self.env['ir.attachment'].sudo()
         for vals in vals_list:
+            vals['is_note'] = vals['subtype_id'][0] == note_id
```

Comment:
"is_message_subtype_note" ?

Also, just to be safe: `message_subtype_note_id and vals.get('subtype_id', [False])[0] == message_subtype_note_id` ?